### ISSUE TITLE
"Applied Visual Design: Adjust the Hue of a Color" does not accept correct answer

### ISSUE THREAD

**Issue by andikay** [freeCodeCamp/freeCodeCamp#38527]
**Describe your problem and how to reproduce it:**
The hsl for cyan is not accepted as the correct answer. 
.cyan {
    background-color: hsl(180, 100%, 50%);
  }
Result: The div element with class cyan should have a background-color of cyan.

I copied the correct answer from the "Get a Hint" page into the code window just to be sure - same result, cyan is not accepted as the correct answer.

**Add a Link to the page with the problem:**
https://www.freecodecamp.org/learn/responsive-web-design/applied-visual-design/adjust-the-hue-of-a-color

**Tell us about your browser and operating system:**
* Browser Name: Pale Moon ( https://www.palemoon.org/ )
* Browser Version: 28.9.1 32bit
* Operating System: Win7 x64

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
![image](https://user-images.githubusercontent.com/23010469/79044662-a2ba7c80-7c06-11ea-9f7f-8b053e6c5e74.png)

**Comment by Sky020** [comment]
Hello, @andikay .

This type of problem is, generally, first discussed on the [freeCodeCamp forum](https://www.freecodecamp.org/forum), by using the _Ask for Help_ button on the referenced lesson's page.

The reason for this is, your issue is most likely to do with your browser, and not the source code here. Either, Pale Moon does not support certain CSS types, or (most likely) you have a setting/extension that is altering the properties of the fCC challenge page.

For this reason, we recommend you use Chrome's latest stable version as your browser, when completing challenges on freeCodeCamp.

**Comment by andikay** [comment]
Okay, good to know. I did see some issues here like mine, so I assumed that is the way to handle it. My bad.

I assume it has something to do with the browser - however, I had no problem completing ALL other challenges so far, also ones that included the colors teal and cyan. Therefore, I have a hard time believing that it is only the browser's fault. I have no extension activated that messes with the CSS. Anyway, I went to Firefox for just this one challenge and it worked. Still a weird behavior in my opinion.
